app/src/main/java/com/powerhungers/fomezero/feature/login/domain/model/User.kt

```diff
@@ -10,4 +10,7 @@ data class User(
     val cpf: String,
     val createdAt: String,
     val updatedAt: String,
-)
\ No newline at end of file
+    val token: String,
+    val refresh_token : String
```

Comment:
```suggestion
    val refreshToken : String
```
o padrão de variaveis no android é camelCase e não snake_case como vem do backend